lib/datasource/remote/model/profile_model.dart

```diff
@@ -100,7 +100,7 @@ class ProfileModel extends Equatable {
   factory ProfileModel.fromJson(Map<String, dynamic> json) {
     return ProfileModel(
       account: hasValue<String>(_account, json),
-      status: enumFromString<ProfileStatus>(ProfileStatus.values, hasEmptyValue<String>(_status, json)),
+      status: enumFromString<ProfileStatus>(ProfileStatus.values, hasEmptyValue<String>(_status, json) ?? ''),
```

Comment:
@n13  Same here, all these fields will never be null? Do they all have a default value? If so, it would save me all these annoying checks.

<img width="692" alt="image" src="https://user-images.githubusercontent.com/42857405/144734534-59a96279-3eef-463c-999e-c977995e93a3.png">

Reply:
Yes the results of chain calls are never null, if the field is not set, it is ""

I am not sure about numbers - as JSON sometimes has numbers - I don't know what numbers are set to if they are undefined. May be 0. It may be that there are no undefined numbers in EOSIO, since all data is default initialized to something.